hacktoberfest/gradlew

```diff
@@ -2,6 +2,7 @@
 
 #
 # Copyright 2015 the original author or authors.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I'm not sure, but isn't this generated code? Shouldn't we keep the file unchanged in this case?

Reply:
Phew, me neither. I am not familiar enough with gradle. One of our internal repos had a static gradlew file so I thought this is code like any other. I seems though that you [generate the file once](https://stackoverflow.com/questions/25769536/how-when-to-generate-gradle-wrapper-files) and can regenerate it once you change the gradle version.

We could of course create a `gradlew.license` file for it, but do we know for certain that the generated file will always be Apache-2.0?

Reply:
Looks like this is something which should be fixed upstream, so that it generates the correct SPDX information.

For now I would be fine with changing the local file so that the linter is happy. And in case the file is automatically regenerated, we would also notice any license changes, because we would have to readd the header.

Reply:
True, I take it in my backlog somewhere.

So we keep this file as it is right now.